packages/schema-generator/src/generate_schema.test.ts

```diff
@@ -20,7 +20,7 @@ void describe('SchemaGenerator', () => {
   void it('should generate schema', async () => {
     const schemaGenerator = new SchemaGenerator();
     await schemaGenerator.generate({
-      connectionString: 'mysql://user:password@hostname:3306/db',
+      connectionUri: 'mysql://user:password@hostname:3306/db',
```

Comment:
is `connectionUri` right terminology ? I.e. are postre and mysql use that term interchangebly with `connetionString` ?

Reply:
That's the feedback from recent API review.